go/libkb/loaduser.go

```diff
@@ -46,6 +46,12 @@ func NewLoadUserByUIDArg(g *GlobalContext, uid keybase1.UID) LoadUserArg {
 	return arg
 }
 
+func (g *GlobalContext) LoadUser(uid keybase1.UID) (*User, error) {
```

Comment:
k.. moved

I can go further and refactor more, just wanted the goal of this PR to keep GlobalContext out of new packages. You could make a `chat.KeybaseContext` impl that wasn't directly GlobalContext to decrease the bloat there. That would be a good thing too :)